crates/fluvio/src/config/cluster.rs

```diff
@@ -14,7 +14,7 @@ use super::ConfigFile;
 /// This is part of profile
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[non_exhaustive]
-pub struct FluvioConfig {
+pub struct FluvioClusterConfig {
```

Comment:
To make it compatible without bumping up major version, add deprecated alias:
```
pub type FluvioConfig = FluvioClusterConfig;
```

Reply:
make sense, done!